src/components/AstronautsPage/actions/getAstronautsData.ts

```diff
@@ -0,0 +1,20 @@
+import { AppDispatch } from '../../../store';
+import type { Astronauts } from '../types/Astronauts';
+import { fetchAstronauts } from '../api/services/fetchAstronauts';
+import {
+  astronautsRequestAction,
+  astronautsSuccessAction,
+  astronautseErorAction,
+} from './actionCreators';
+
+export const getAstronautsData =
+  () =>
+  async (dispatch: AppDispatch): Promise<void> => {
```

Comment:
the same about AppDispatch
and why getAstronautsData in actions folder

Reply:
Where this file should be? In the services?